javascript/ecmascript-2015/destructuring/defaults-values-apply-only-to-undefined-and-not-to-null.md

```diff
@@ -78,9 +86,7 @@ const {a = 1} = {a: undefined}
 ## Revision
 
 ```javascript
-function bar(a = 1) {return a}
-//foo(null) === null
-const {a = 1} = {a: null}
+const { a = 1 } = { a: null };
```

Comment:
Weird that this revision question doesn't have text. Can we maybe copy the same text from the practice question?

Reply:
done

Reply:
done